### PR TITLE
Unsubscribe page updates

### PR DESCRIPTION
**Pull request by sukhada**
Covers issues listed here: https://github.com/350org/ak_intl_v3/issues/332

### CODE REVIEW

**Comment by rachelhbd** [comment]
hey folks, just bumping this one - can it be reviewed and merged into Develop by EOD tomorrow so I can QA this on Weds? Or today is even better! Thanks much

**Review by cjmabry — COMMENTED** [review]
@sukhada @rachelhbd  I looked this over and it seems good but have a few questions, mostly about what the scope of these changes are and what the expectation for code review is.

- for the parts that replace the autoescape block with the include_tmpl tag, that makes sense and looks good.
- for the thanks_unsubscribe page and the unsubscribe page, it looks like these are wholesale copies from the FA branch here (https://github.com/350org/ak_intl_v3/pull/341/files) with no additional changes from WTR - is that correct? If so, was the expectation that I should fully review those changes from FA as well, or just ensure they were copied over correctly? If the former, I'm less sure of the context and scope of these original changes from FA so would probably take a little longer to fully review those. If the latter, I've verified these are 1 to 1 copies of FA changes into this template set.
- For the thanks page, I believe these include changes both from WTR and FA, so it's pretty hard to follow who made what changes. Can you advise on how we should review this page? Are we trusting FA changes have been code reviewed and I just need to look over the WTR work and integration? Or was the expectation to fully review the entirety of these combined changes? If the former, is there a way to stage the PRs so that I can review the WTR work separately form integrating the FA work (or perhaps that has alreday been done and I'm just checking these off for any loose ends?)

**Comment by rachelhbd** [comment]
thanks Chris! the work @sukhada did was originally here: https://github.com/350org/ak_intl_v3/pull/351
With this language fix added later: https://github.com/350org/ak_intl_v3/pull/372

Sukhada can confirm, but you might not be seeing all the changes because PR 351 got merged *into* the FA branch already?

Thanks for this, sounds good!
> I've verified these are 1 to 1 copies of FA changes into this template set.
I'm okay with this light review, checking the merge matches - because Alan already reviewed 351 and 372, and because it's been a complex merge I'm planning to do a full QA of the feature to check the functionality. Sukhada does that sound right to you?

If there's a risk of overwriting any more recent code that might affect any other features that would be worth further review, but looking at the code difs it looks like Sukhada's already taken care of this and removed any conflicts that aren't related to the unsub feature.

**Comment by cjmabry** [comment]
@rachelhbd That sounds good! I'll mark this as approved from my end, and I agree thorough QA would be a good idea here. Let me know if you need help on that front.

**Review by cjmabry — APPROVED** [review]

**Comment by rachelhbd** [comment]
hey @sukhada @cjmabry I've [done a round of QA](https://docs.google.com/spreadsheets/d/1LNa5Jnani1g_1YmJDFCHzVj2esJND9NadmPxQL3Z3Ss/edit#gid=1703017732). I didn't find any browser/device issues, but did find:
-  **a bug with the "unsubscribe from all" link** - perhaps a JS issue of some sort?
This feature does work in the [original FA version](https://act.350.org/act/unsubscribe-new/?akid=.5148234.PNjCJz&template_set=350.org%20-%20International%20v3%20(FA%20sprint%20nov%2021)) but not in Develop or the templateset using the new wtr-fa-unsubscribe-page branch
Also I was wondering, if the link does rely on javascript rather than html, do you think having an unsub submit that relies on JS is robust enough here? This is a link we should be sure should work for all users. We could consider reviewing this as part of the Sept sprint - maybe we can have this as a use case for an automated test)

- **survey responses are being recorded as a sentence** - sorry I didn't spot that in FA's work before, it would be better for the values to be simplified so it's clear they include multilingual responses and allows us to tweak phrasing (which we've already had requests for)
This one is non-blocking and we could update in Sept, but might be a relatively quick fix and would keep our data in a better shape
The current parent field name is good (survey_unsubscribe_reason), for the values we could use the names from the language strings? eg. "frequency" "consent" etc? Here is how they match up:

> frequency=You send me too many emails
> consent=I never signed up for this mailing list
> difemail=I signed up with a different email
> campaigns=I'm no longer interested in 350.org's campaigns
> other=Other
